synchronizer/l1_workers.go

```diff
@@ -84,15 +91,15 @@ func (w *workers) initialize() error {
 }
 
 func (w *workers) stop() {
-	log.Debugf("workers: stopping workers %s", w.String())
+	log.Infof("workers: stopping workers %s", w.String())
 	for i := range w.workers {
 		wd := &w.workers[i]
 		if !wd.worker.isIdle() {
 			log.Debugf("workers: stopping worker[%d] %s", i, wd.String())
 		}
 		wd.ctx.cancel()
 	}
-	w.waitFinishAllWorkers()
+	//w.waitFinishAllWorkers()
```

Comment:
Why don't you wait here?

Reply:
Because you don't need to wait to finish all workers, the workers are going to be canceled eventually and you can go on with the regular flow. Just to speed up process